client/src/components/MuiFooter.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {
+    // Container,
+    Grid,
+    Box,
+    Button
+ } from '@mui/material';
+import  Link  from '@mui/material/Link';
+import FacebookIcon from '@mui/icons-material/Facebook';
+import InstagramIcon from '@mui/icons-material/Instagram';
+import TwitterIcon from '@mui/icons-material/Twitter';
+import GitHubIcon from '@mui/icons-material/GitHub';
+import LinkedInIcon from '@mui/icons-material/LinkedIn';
+// import LinkTree from './LinkTree';
```

Comment:
if we are not using this, let's remove this as well.

Reply:
Removed old footer component.